### ISSUE TITLE
Add syntax highlighting for match-case clauses (PEP 634, 635, 636)

### ISSUE THREAD

**Issue by idanw206** [ipython/ipython#13046]
I tried the new feature in Python 3.10.0b3 using IPython, and the syntax wasn't highlighted.
I think that it will be great to support it also for the RC versions.

Thanks!

references:
https://www.python.org/dev/peps/pep-0634
https://www.python.org/dev/peps/pep-0635
https://www.python.org/dev/peps/pep-0636

**Comment by MrMino** [comment]
Yup, this is yet to happen. Relevant module (AFAIK) for people who want to help: [`IPython.utils.PyColorzie`](https://github.com/ipython/ipython/blob/master/IPython/utils/PyColorize.py), which is also used by `%pycat`.

**Comment by 12somyasahu** [comment]
can i take it?

**Comment by Carreau** [comment]
Don't ask, just send a PR.

**Comment by 12somyasahu** [comment]
> Don't ask, just send a PR.

Thanks! Tested on IPython 8.38.0 with Python 3.10 — match/case is already highlighted correctly. Should this issue be closed?
Also, I'd love to contribute more. Any beginner-friendly issues you'd point me to?

**Comment by Carreau** [comment]
You can have a look at https://github.com/ipython/ipython/issues/14280; and investigate if the function can be actually removed, or marked as removal for later; 

And yes, if this is working now; we can close.
